xarray/tests/test_dataarray.py

```diff
@@ -2544,4 +2530,3 @@ def test_rolling_reduce(da, center, min_periods, window, name):
     actual = rolling_obj.reduce(getattr(np, 'nan%s' % name))
     expected = getattr(rolling_obj, name)()
     assert_allclose(actual, expected)
-    assert da.dims == expected.dims
```

Comment:
Good point, I missed that this was the xarray function and not the numpy one. So it is indeed redundant (but harmless).